no-computer.md

```diff
@@ -58,12 +86,23 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think that this project is pretty much 100% language design.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+Hmmm I'm not sure.  I think it has a relatively wide scope since there are many
+physical attributes that define people's appearence and that these can look very
+similar with slight differences.  There would be design decisions about what level
+of detail would be useful for describing a person vs what level of detail would
+be too intricate for a listener to really remember everything and differentiate
+each detail.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+It would be a good project because it has pretty much just design choices to make
```

Comment:
It's an interesting idea, but I think @wholien had it right when he said that the best DSL, photos, already exist.